cmds/core/shutdown/shutdown_linux.go

```diff
@@ -43,25 +44,26 @@ var (
 	}
 )
 
-func usage() {
-	fmt.Println("shutdown [<-h|-r|-s|halt|reboot|suspend> [time [message...]]]")
-}
-
-func shutdown(args []string, dryrun bool) (uint, error) {
+// Shutdown calls unix.Reboot, with the type of shutdown defined in args, currently
+// halt, reboot, or suspend. A time may be specified as "now",
+// a future time parseable by time.ParseDuration, or in
+// RFC3339 format. If dryrun is chosen, Shutdown returns the opcode it
+// would have used and an error, if any.
+func Shutdown(dryrun bool, args ...string) (uint, error) {
```

Comment:
Need this be a public ? -- it is not a pkg, and not referenced anywhere except from test. shutdown_linux_test is in the same `main`, should be able to use `shutdown` as private

Reply:
it need not be public, but otoh, it kind of points to maybe making this a package someday.

Reply:
I went ahead and changed it back to shutdown.